python/tests/unit/arcticdb/version_store/test_aggregation.py

```diff
@@ -23,7 +23,10 @@
 from hypothesis import assume, given, settings
 from hypothesis.extra.pandas import column, data_frames, range_indexes
 
+from tests.util.mark import LINUX_CONDA_MARK
 
+
+@LINUX_CONDA_MARK
```

Comment:
Any ideas what's going wrong with this one? We should definitely at least have a ticket to come back and fix it...

Reply:
I have no idea, I try to debug it next week when I have better access to conda on the new laptops.
I will create an issue about this in the mean time.